clio-server/src/main/scala/org/broadinstitute/clio/server/service/SearchService.scala

```diff
@@ -15,6 +16,21 @@ import org.broadinstitute.clio.transfer.model.ClioIndex
   */
 class SearchService private[server] (searchDAO: SearchDAO) {
 
+  /**
+    * Run a simple string query.
+    * @param simpleString The simple json string to query.
+    * @tparam CI The type of the TransferKey to query.
+    * @return The result of the query.
+    */
+  def simpleStringQueryMetadata[Input, CI <: ClioIndex](simpleString: String)(
```

Comment:
Would it make sense for this to take `Json`, and handle the stringification? The fact that it's needed as a `String` feels like a detail of `SimpleStringQueryDefinition` that might be worth hiding.